weaver/sdks/fabric/interoperation-node-sdk/.gitignore

```diff
@@ -4,8 +4,5 @@ node_modules/
 coverage/
 package-lock.json
 old/
-protos/
-protos-js/*
```

Comment:
Any reason for this change?
Also I see there is no `protos-js` in SDK, so I guess you figured out building SDK without checking in `protos-js` in SDK?

Reply:
This was reverted.